ampproject/amp-wp
# Move adding background media to the toolbar

Fixes: #2699

Steps to test: 
1. Create a new story. 
1. Click on media icon in top toolbar. 
1. Dropdown should appear with following options.
   - Insert image (inserts block)
   - Insert video (inserts block)
   - Insert background image (opens media modal)
   - Insert background video (opens media modal)
1. Insert image / video should these blocks into the current page. 
1. Insert background image / video, should open a media picker dialog and should allow for upload / setting of background.
